server/controllers/stock/core.js

```diff
@@ -428,9 +428,13 @@ async function getDailyStockConsumption(params) {
   const filters = new FilterParser(params, { tableAlias : 'm' });
 
   const sql = `
-    SELECT SUM(m.quantity) as quantity, DATE(m.date) as date, 
-        i.uuid AS inventoty_uuid, i.text AS inventory_name,
-        d.text AS depot_name, d.uuid AS depot_uuid, m.period_id
+    SELECT SUM(m.quantity) as quantity,
+      SUM(m.quantity*m.unit_cost) as value,
+      DATE(m.date) as date,
+      HEX(i.uuid) AS inventory_uuid,
+      i.text AS inventory_name,
+      d.text AS depot_name,
+      HEX(d.uuid) AS depot_uuid
```

Comment:
Could you use `BUID()` here instead of `HEX()`?  They are the same thing at the moment, but it was a pretty painful migration back in 1.x when we had to change the format and everyone had written their own custom code.

Reply:
okay